contrib/hashicorp/consul/consul.go

```diff
@@ -81,6 +91,11 @@ func (k *KV) startSpan(resourceName string, key string) ddtrace.Span {
 		tracer.Tag(ext.SpanKind, ext.SpanKindClient),
 		tracer.Tag(ext.DBSystem, ext.DBSystemConsulKV),
 	}
+
+	if k.config.consulConfig != nil && k.config.hostname != "" {
```

Comment:
We're not actually using `consulConfig` here. Can we just check that `hostname != ""`?